packages/jest-core/src/TestScheduler.ts

```diff
@@ -28,6 +29,7 @@ import {
   buildFailureTestResult,
   makeEmptyAggregatedTestResult,
 } from '@jest/test-result';
+import interopRequireDefault from 'jest-util/build/interopRequireDefault';
```

Comment:
The pnp error is very obscure, but *maybe* it doesn't like this?
(Importing from somewhere inside another package, which we always avoid, so should change this anyway)

Reply:
If this is it, there really should be a better error

(and yeah, should fix this import. Wonder if there's some lint rule we can add?)

Reply:
VSCode's auto import messed up the path. I fixed the import, no luck in the test